### ISSUE TITLE
00:00 elapsed time

### ISSUE THREAD

**Issue by ZeroV1p3r** [Stoozy/vimcord#9]
hello, i used vimcord and it works fine
but ihave an issue the elapsed time work just in my account and my friends see rpc work fine but time elapsed stoped in 00:00 also in my phone when i checked 
or how i can disable elapsed time for showing

**Comment by Stoozy** [comment]
That's really weird. I don't know why it would just stop at 0:00. I don't think just not displaying it would be a good solution. How long did it stay at 0:00? Have you tried opening different files?

**Comment by ZeroV1p3r** [comment]
> That's really weird. I don't know why it would just stop at 0:00. I don't think just not displaying it would be a good solution. How long did it stay at 0:00? Have you tried opening different files?

yes that's also a solution but i don't know how i can not displaying it, i tried a lot of files i edited 6 hours and also stay 00:00, thanks a lot for your fast answer.

**Comment by Stoozy** [comment]
I'll have to look more into this.
